app/utilities/upload_types/upload_requirements.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class UploadRequirements
+  class << self
+    def requirements_messages(type)
+      [validation_messages_presence(type),
+       validation_messages_numericality(type),
+       validation_messages_uniqueness(type)]
+        .compact
+    end
+
+    def validation_messages_inclusion(type)
+      inclusion = type.validators.map do |validations|
+        next unless validations.class == ActiveModel::Validations::InclusionValidator
+
+        { message: affected_attributes(validations, type).join(', '),
+          value: inclusion_requirement_message(validations) }
+      end
+      inclusion unless inclusion.empty?
```

Comment:
```suggestion
      type.validators.map do |validations|
        next unless validations.class == ActiveModel::Validations::InclusionValidator

        { message: affected_attributes(validations, type).join(', '),
          value: inclusion_requirement_message(validations) }
      end.presence
```

You could also use `presence` to check emptiness.

Reply:
Switched to using `presence`, map wasn't working correct so switched to each